.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
[OPTIONAL] I suggest you add this to the .gitignore file This represents the local settings for your vscode.